src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/step/allocation/AttemptAllocationStep.kt

```diff
@@ -0,0 +1,73 @@
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.step.allocation
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.suspendUntil
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.ManagedIndexMetaData
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.action.AllocationActionConfig
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.managedindexmetadata.StepMetaData
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.step.Step
+import org.apache.logging.log4j.LogManager
+import org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequest
+import org.elasticsearch.action.support.master.AcknowledgedResponse
+import org.elasticsearch.client.Client
+import org.elasticsearch.cluster.service.ClusterService
+import org.elasticsearch.common.settings.Settings
+
+class AttemptAllocationStep(
```

Comment:
Also saw the mappings had a wait_for which I'm assuming is a flag to optionally wait for the shards to finish migrating to their respective filtered nodes, but I don't see it used. Are we skipping it for the first implementation? That's fine if so, otherwise we would need another step (WaitForAllocation or something) that would periodically check the shard locations.

Reply:
That's correct, it's a placeholder for future step.